### PR TITLE
Abe21, Diez21, Pannus21, Riou21, Sahin21

### PR DESCRIPTION
**Pull request by shrutipai**

### CODE REVIEW

File: tables/subjects.csv [hunk 1]
```diff
@@ -1,4 +1,8 @@
-ref_name,subject_name,subject_species,birth_year,num_subjects
+﻿ref_name,subject_name,subject_species,birth_year,num_subjects
+Abe21,Resident_Moderna,Human,NULL,1
```
Comment (KaimingTao) [inline tables/subjects.csv]: please update subject number

File: tables/subjects.csv [hunk 2]
```diff
@@ -1153,6 +1153,10 @@ Moyo-Gwete21,CP_B.1.351,Human,NULL,1
 Muecksch21b,CP_SF4,Human,NULL,1
 Muik21,BNT162b2_d29,Human,NULL,1
 Muik21,BNT162b2_d43,Human,NULL,1
+Pannus21,Infected_Resident,Human,NULL,1
```
Comment (KaimingTao) [inline tables/subjects.csv]: please update the subject number

Comment (shrutipai) [reply tables/subjects.csv]: Hi Kaiming, I responded to one of your earlier comments as well but I actually edited the subject number for all the papers all at once in one of the later commits, with this sha id: daa90fe1ad88d9e57b5eb844c33a90077ad9a503

Comment (KaimingTao) [reply tables/subjects.csv]: OK, sorry I just go from the top one.

File: tables/isolate_mutations.csv [hunk 3]
```diff
@@ -4490,3 +4490,13 @@ B.1.351 Spike:-18F+241del-244del-246I,S,484,K
 B.1.351 Spike:-18F+241del-244del-246I,S,501,Y
 B.1.351 Spike:-18F+241del-244del-246I,S,614,G
 B.1.351 Spike:-18F+241del-244del-246I,S,701,V
+B.1.351 Spike:-246I-614G,S,18,F
```
Comment (KaimingTao) [inline tables/isolate_mutations.csv:4493]: For pseudovirus, we prefer to use whole Spike mutation as the iso_name, like S:18F+501Y+614G. You can change B.1.351 Spike:-246I-614G to the "S form".

File: tables/ref_isolate_pairs/riou21-pair.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+﻿ref_name,control_iso_name,iso_name
+Riou21,S:614G,B.1.351 Spike:-246I-614G
+Riou21,B.1.351 Spike:-246I-614G,S:614G
```
Comment (KaimingTao) [inline tables/ref_isolate_pairs/riou21-pair.csv:3]: This is correct, you should add this row. The CP_Wave2 was about Beta variant infection. The database check will make sure the infection matchs the control_iso_name. That's why the program throws error when this row is missing.

**Pull request merged**